Scripts/sort-schemes.sh

```diff
@@ -0,0 +1,119 @@
+#!/bin/bash
+
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+# Sort the order of projects' schemes.
+# Usage: sort-schemes.sh
+
+# Alias for replacing symbols.
+aliasNewLine='@'
+aliasNumber='%numberAlias%'
+
+# Regex constans for change number.
+regexStartPlaceNumber="<key>[a-zA-z -^#.]*<\/key>[${aliasNewLine}][[:space:]]*<dict>[${aliasNewLine}][[:space:]]*<key>orderHint<\/key>[${aliasNewLine}][[:space:]]*<integer>"
+regexEndPlaceNumber="<\/integer>[${aliasNewLine}][[:space:]]*<\/dict>"
+
+# Regex constans for hidden schemes.
+regexHiddenScheme="<key>isShown<\/key><false\/>"
```

Comment:
if `isShown` explicitly set to `true`, this script doesn't work, not sure that it's a blocker